app/uk/gov/hmrc/hec/services/TaxCheckService.scala

```diff
@@ -42,7 +41,7 @@ trait TaxCheckService {
     hc: HeaderCarrier
   ): EitherT[Future, Error, HECTaxCheckMatchResult]
 
-  def getUnexpiredTaxCheckCodes(ggCredId: GGCredId, today: LocalDate)(implicit
+  def getUnexpiredTaxCheckCodes(ggCredId: GGCredId)(implicit
```

Comment:
any possibility we can rename to getUnexpiredTaxCheckCodesByCredId? or will it be too long for a name?

Reply:
don't think it's super long, but feels a bit repetitive since the param name conveys the intent

Reply:
it's always nice if function name depicts that, but not a big deal . resolving